### DISCUSSION TITLE
Do not use formalities ...  @dpolivaev

### DISCUSSION THREAD

**Discussion by theelectricsoul** [freeplane/freeplane#1023] [Help]
@dpolivaev 
I am not sure if I offended you :) 

Referring to
https://github.com/freeplane/freeplane/issues/980#issuecomment-1426759274
>I know you are the user. I just do not like formalities.

Sure in this case....   there are no more roles. It is just a habit of mine. In our company there are lots of roles. 

>"I think behavior X is not consistent with my expectations. I expect X because of ... but observe Y"

Because you wrote that line, I assume you could think that I am trying to be a wise knowing everything jackass :) If so, not intentionally...lol.

[Let me explain :)](https://user-images.githubusercontent.com/10260832/221187920-d89d4357-af4a-4c35-9c91-004abec313b1.mp4)

If I have hit the nail on the head... I will rephrase in the future :)

**Comment by dpolivaev** [comment]
I find your issues so well done and nearly perfect that I could not resist my wish to suggest an improvement.
The rest is up to you.
